samples/libffmpeg/LibffmpegMain.java

```diff
@@ -32,14 +32,14 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import jdk.incubator.foreign.*;
+import java.lang.foreign.*;
 import libffmpeg.AVCodecContext;
 import libffmpeg.AVFormatContext;
 import libffmpeg.AVFrame;
 import libffmpeg.AVPacket;
 import libffmpeg.AVStream;
 import static libffmpeg.Libffmpeg.*;
-import static jdk.incubator.foreign.MemoryAddress.*;
+import static java.lang.foreign.MemoryAddress.*;
```

Comment:
Most of other samples statically import just NULL

Reply:
Will fix with specific import